lib/languages/languages.dart

```diff
@@ -0,0 +1,4 @@
+/// Barrel file for Languages barrel files
+/// Please sort alphabetically
+
+export 'kor/kor.dart';
```

Comment:
Would we go with Alpha-3 code?

Ref: https://www.iban.com/country-codes

Reply:
I'm not so sure, will country codes cover all languages? 
I'm afraid that language is more comprehensive than country.

ex)
<img width="687" alt="스크린샷 2022-10-07 오후 4 33 57" src="https://user-images.githubusercontent.com/54804279/194498848-e8d29eec-41c3-4614-9153-1a3cde45368c.png">
<img width="679" alt="스크린샷 2022-10-07 오후 4 34 18" src="https://user-images.githubusercontent.com/54804279/194498854-df2ac62e-5a60-4aff-a51d-4242105ffedf.png">

Reply:
We can choose like below
- KR ( only Alpha 2 code )
- KOR ( only Alpha 3 code )
- KR_KOR ( combine Alpha 2, 3 code )

kr.dart
kor.dart
kr_kor.dart

Reply:
I don't think the suggested methods do not meet my question. 
My question was "will country codes cover all languages?"

For example, if I want to add Latin(a language that no country uses anymore), will it be able to express that in the boundary of country codes? Or a language that minorities use?

Reply:
Oh I got it.

We should find out another package with l10n, then just opt to follow it!

Reply:
Here's [language list supported by Google Cloud Translation](https://cloud.google.com/translate/docs/languages)
[This one](https://github.com/michaelwittig/node-i18n-iso-countries) shows that i18n basically uses ISO-639-1 and others if necessary.

I think we should use the ISO code name, but we can still discuss what versions to choose.
What I've suggested is `639-2/T` in the `Reference` link below.

## References
- [compared list between various language code name systems](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)

Reply:
BTW, I've just noticed, according to [this](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes), ISO 639-2 is Alpha-3!
https://www.loc.gov/standards/iso639-2/php/code_list.php
<img width="704" alt="스크린샷 2022-10-07 오후 6 44 43" src="https://user-images.githubusercontent.com/54804279/194525465-d0d39c6b-dc90-45a5-aac3-c8bdd48748b2.png">

Reply:
I like your example!

I think It will be good with ISO Standard, 639-1, 639-2 ..
You can choose whatever you want!